roles/setup_tftp/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Setup TFTP
+  block:
+    - name: Install TFTP
+      ansible.builtin.package:
+        name: tftp-server
+        state: present
+
+
+    - name: Start TFTP
+      ansible.builtin.service:
+        name: tftp.socket 
+        state: started
+        enabled: true
```

Comment:
You already do this in boot_iso

Reply:
yes I moved that from there this is the draft file